tools/run-dev.py

```diff
@@ -77,7 +77,7 @@
         # In the Vagrant development environment, we need to listen on
```

Comment:
this should check for `zulipdev` as well for the Digital Ocean dev environment setup.